README.md

```diff
@@ -173,13 +172,14 @@ Feel free to check the [issues page](../../issues/).
 
 > I would like to thanks and give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), for the design inspiration as required by the [Creative Commons License](https://creativecommons.org/licenses/)
 
+>I would like to thanks and give appropriate Microverse for this huge opportunity  [My Loom Video](https://www.loom.com/share/b78e8da03e3d47e7bb1b4c16008f15a8/) 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly as stated by the previous reviewer 😃 Good job with the project presentation, however, your missed in your presentation to highlight a piece of the code (perhaps from your code editor) that you find interesting or that you were proud of making. Let's include this missing piece in the video presentation.💯👌